app/components/RadioPlayer.vue

```diff
@@ -566,31 +566,15 @@ export default {
       this.streamModal = false
     },
     getLatestEpisodeFromArcsi() {
-    // TODO: use dedicated frontend API instead of one meant for arcsi UI (rationale: we need the unarchived episodes too for relay logic and there was no such frontend arcsi API available)  
-    // TODO: see also related workaround in getLatestEpisode ()
-    this.$axios.get(arcsiBaseURL + '/show/' + this.show.id, config)
+    this.$axios.get(arcsiBaseURL + '/show/' + this.slugify(this.show.name) + '/page?filter=latest', config)
```

Comment:
Here we want to return the latest `item` for all the episodes of the `show` (including not yet aired episodes), right?
Just because there is a workaround in the Arcsi right now which adds the `archived` flag as "true" for all the requests coming from the frontend (checks if the origin URL contains `https://www.lahmacun.hu`)

Reply:
Ok, reverted my original commit on this file. @tuz666 please check.